backend_site/posts/views.py

```diff
@@ -0,0 +1,26 @@
+from flask import Blueprint, jsonify
```

Comment:
This was initially made as an example API endpoint. This isn't used anywhere on the front end as it is now IIRC.

Reply:
Ok, I'll cut it